entries/transfer.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<entry name="transfer" type="method" return="jQuery">
+	<title>.transfer()</title>
+	<desc>Transfers the outline of an element to another element</desc>
+	<longdesc>
+		<p>Very useful when trying to visualize interaction between two elements.</p>
+		<p>The transfer element itself has the class <code>ui-effects-transfer</code>, and needs to be styled by you, for example by adding a background or border.</p>
+	</longdesc>
+	<signature>
+		<argument name="options" type="Object">
+			<property name="to" type="Selector">
+				<desc>The target of the transfer effect.</desc>
+			</property>
+			<property name="className" type="String" default="null">
+				<desc>A class to add to the transfer element, in addition to <code>ui-effects-transfer</code>.</desc>
+			</property>
+			<property name="duration" type="Number" default="400">
```

Comment:
Use the content from https://github.com/jquery/api.jqueryui.com/blob/master/includes/animation-argument-duration.xml.